### DISCUSSION TITLE
Hide rooms in map

### DISCUSSION THREAD

**Discussion by martindybal** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#739] [General]
I know it's a silly question. But is there a way to hide a room from the map?

As you can see, there is a lot of outside view (room 7 and 11). Can I hide them to see the map bigger?

![image](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/assets/7644099/db595808-c9c3-4080-9cea-3870c24a2f6e)

**Comment by PiotrMachowski** [comment]
Optimally it should be handled at the integration level, but you can use [cropping functionality](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card?tab=readme-ov-file#cropping-options) to adjust visible part of the map (it does not always work perfectly at this moment though)

**Reply by martindybal** [reply]
Thanks a lot!
